runtime/domain.c

```diff
@@ -45,20 +45,26 @@
 #include "caml/sync.h"
 #include "caml/weak.h"
 
-/* From a runtime perspective, domains must handle stop-the-world (STW)
-   sections, during which:
-    - they are within a section no mutator code is running
-    - all domains will execute the section in parallel
-    - barriers are provided to know all domains have reached the
-      same stage within a section
+/* The runtime can run stop-the-world (STW) sections, during which all
+   active domains run the same code in parallel (with a barrier
```

Comment:
Active domains are the same as STW participants?

Reply:
In my mind "active domain" is an informal term to mean "domains currently running", in particular all domains that may run OCaml code ("mutator code"). This is the comment at the top of the file, it's giving a high-level description only, and at this point readers don't really know what "STW participants" are.